pkg/invoke/invoker.go

```diff
@@ -184,6 +184,7 @@ type Options struct {
 	CredentialContextIDs  []string
 	UserUID               string
 	GenerateName          string
+	ParentThreadName      string
```

Comment:
If you create cred directly in ui, the api is called in a different route which don't have parent thread(project thread) available. So this is to pass the parent thread